dbt_semantic_interfaces/dbt_semantic_interfaces/object_utils.py

```diff
@@ -0,0 +1,104 @@
+from __future__ import annotations
```

Comment:
While we're moving this file can we rename it to `pretty_print.py` or similar, since that's all this module does now? Then we'll be rid of object_bag_of_crap.py once and for all!

Reply:
Absolutely!